### PR TITLE
libssh 0.9.5

### LINKED ISSUE #23: Update to libssh 0.9.5
libssh 0.9.x up to 0.9.5 contains a bug where ssh_channel_read() will sometimes incorrectly return SSH_ERROR. This happens a lot when executing short-lived programs that produce no or little output.

- libssh bug: https://bugs.libssh.org/T231
- libssh change: https://gitlab.com/libssh/libssh-mirror/-/commit/750e4f3f9d3ec879929801d65a500ec3ad84ff67

(eref 42d478fb6976)

### PR DESCRIPTION
**Pull request by enkore**
Fixes #23

(eref 42d478fb6976)

### CODE REVIEW

**Comment by enkore** [comment]
Building, installing & using a wheel locally works. As far as I can tell this doesn't change requirements. Don't know what causes the CI failures, CircleCI unfortunately requests write access to all my repositories to let me look at the logs.

File: setup.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
 runtime_library_dirs = ["$ORIGIN/."] if not SYSTEM_LIBSSH else None
 _lib_dir = os.path.abspath("./src/lib") if not SYSTEM_LIBSSH else "/usr/local/lib"
-include_dirs = ["libssh/include"] if (ON_WINDOWS or ON_RTD) or \
+include_dirs = ["libssh/include", "./src/include"] if (ON_WINDOWS or ON_RTD) or \
```
Comment (pkittenis) [inline setup.py:67]: Is this change needed for something? `libssh/include` should already have the correct include file.

**Review by pkittenis — CHANGES_REQUESTED** [review]
This [check](https://github.com/ParallelSSH/ssh-python/blob/master/tests/test_channel.py#L106) can now be removed - it was added for 0.9.4 and is causing build failures.

The [libssh archive](https://github.com/ParallelSSH/ssh-python/blob/master/ci/docker/manylinux/libssh-0.9.4.tar.xz) used for the manylinux builds will also need updating. It needs to be copied as docker cannot include files outside the dockerfile root dir.

**Comment by pkittenis** [comment]
The [libssh archive](https://github.com/ParallelSSH/ssh-python/blob/master/ci/docker/manylinux/libssh-0.9.4.tar.xz) used for the manylinux builds will also need updating - it needs to be copied as docker cannot include files outside the dockerfile root dir.

**Comment by pkittenis** [comment]
Thanks for raising! Comments above.

File: setup.py (hunk 1 above)
Comment (enkore) [reply setup.py:67]: CMake is now used to generate the libssh_version.h which is placed in the build tree. I suspect a cleaner way to handle this is to do a `cmake --install src --prefix local` after the build and then add `local/include` and `local/lib` as include/library directives. This way, the ssh-python build becomes decoupled from what the libssh CMake file is doing exactly.

**Comment by enkore** [comment]
Wheels seem to build correctly now.

File: setup.py (hunk 1 above)
Comment (pkittenis) [reply setup.py]: Makes sense, thanks for making the change.

**Review by pkittenis — APPROVED** [review]
